app/staking/page.tsx

```diff
@@ -320,14 +330,20 @@ export default function StakingPage() {
                       },
                     ]}
                     content={[
-                      ...userStaking.validators.map(
-                        (userStakingElement, index) =>
+                      ...userStaking.validators
```

Comment:
A better way to do this would be BigNumber(e.userDelegation.balance).isGreaterThan(0). much more accurate.